test/acceptance/tests.py

```diff
@@ -15,8 +16,10 @@
     SubmissionPage, AssessmentPage, GradePage, StaffAreaPage
 )
 
+# This value is generally used in jenkins, but not locally
+PROFILING_ENABLED = os.environ.get('ORA_PROFILING_ENABLED', False)
 
-def retry(tries=4, delay=3, backoff=2):
+def retry(tries=2, delay=4, backoff=2):
```

Comment:
Ah, that explains it then, cool. Thanks for all your help here @benpatterson!

https://openedx.atlassian.net/browse/TNL-4143